src/lib/sql/sql_translator.cpp

```diff
@@ -83,10 +83,8 @@ PredicateCondition get_predicate_condition_for_reverse_order(const PredicateCond
 
 JoinMode translate_join_type_to_join_mode(const hsql::JoinType join_type) {
   static const std::unordered_map<const hsql::JoinType, const JoinMode> join_type_to_mode = {
-      {hsql::kJoinInner, JoinMode::Inner},     {hsql::kJoinOuter, JoinMode::Outer},
-      {hsql::kJoinLeft, JoinMode::Left},       {hsql::kJoinLeftOuter, JoinMode::Left},
-      {hsql::kJoinRight, JoinMode::Right},     {hsql::kJoinRightOuter, JoinMode::Right},
-      {hsql::kJoinNatural, JoinMode::Natural}, {hsql::kJoinCross, JoinMode::Cross},
+      {hsql::kJoinInner, JoinMode::Inner}, {hsql::kJoinFull, JoinMode::Outer},      {hsql::kJoinLeft, JoinMode::Left},
```

Comment:
I don't understand how these changes fit into the PR

Reply:
See the description. When development of this feature started, the current state of sql-parser was not integrated into Hyrise. In detail the changes of this PR: https://github.com/hyrise/sql-parser/pull/81 In order to be able to compile at all, these issues had to be resolved first.

Should I extract these changes into a pull request of its own?

Reply:
No, that's a really small thing so it's fine